docs/configuration/FederateInfo.md

```diff
@@ -1,20 +1,25 @@
 # Federate info
-The FederateInfo structure contains information that can be passed to a federation upon construction. Some information can be updated continuously other can be only be changed before initializationMOde is entered.  
 
+The FederateInfo structure contains information that can be passed to a federation upon construction. Some information can be updated continuously other can be only be changed before initializationMOde is entered.
```

Comment:
Should initializationMOde be initializationMode or initialization mode instead?

Reply:
probably should be initializing Mode to match the 2.0 branch, but we can do that later